projects/OpenMote-CC2538/SConscript.env

```diff
@@ -25,4 +25,8 @@ buildEnv.Append(
    ]
 )
 
+# Specify default crypto engine for the board 
+if not env['cryptoengine']:

```

Comment:
Other boards will use the dummy_crypto_engine since the CRYPTO_ENGINE_SCONS will be undefined (see {{bsp/boards/common/crypto_engine.h}}).